.github/workflows/run-snapshot-test.yml

```diff
@@ -0,0 +1,92 @@
+name: Snapshot verification
+
+on:
+  pull_request:
+    branches: [ main, develop, feature/LD-91/golden-tests ]
+
+permissions:
+  checks: write
+  contents: write
+  statuses: write
+  pull-requests: write
+  actions: write
+
+jobs:
+  test:
+    name: Verify snapshots
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          lfs: true
+
+      - name: Prepare Android Environment
+        uses: ./.github/actions/prepare-android-env
+
+      - name: Gradle - Verify snapshots with Paparazzi
```

Comment:
question: so the CI verifies the difference, but we still have to create screenshots on our own, yes?

Reply:
Yes, recording on CI will be the next thing I handle